### PR TITLE
[next] docs update GLTF and loading Assets pages

### PR DESCRIPTION
**Pull request by DefinitelyMaybe**
helps #686

Will create another for main as well.

### CODE REVIEW

File: apps/docs/src/content/learn/basics/loading-assets.mdx [hunk 1]
```diff
@@ -4,126 +4,35 @@ title: Loading Assets
 order: -500
 ---
 
-A typical Threlte application will make use of textures, models, and other assets. These assets are loaded using the `useLoader` hook. Assets loaded via `useLoader` are automatically cached and will not be loaded or parsed again if they are already in the cache.
+There are a number of different assets you might like to use in your Threlte application. Below are sections to guide you on how to get started with them. Three.js has a [ton of different loaders](https://threejs.org/docs/index.html?q=loader) and Threlte can use all them. You're recommended to use the [useLoader](/docs/reference/core/use-loader) hook to cache assets and for convenience in asynchronous setups. This will help reduces network requests, bandwidth, memory usage and improve the performance of your application.
```
Comment (michealparks) [inline apps/docs/src/content/learn/basics/loading-assets.mdx:7]:
```suggestion
There are a number of different asset types you might like to use in your Threlte application. Three.js has a [ton of different loaders](https://threejs.org/docs/index.html?q=loader) that Threlte integrates well with. You're recommended to use the [useLoader](/docs/reference/core/use-loader) hook which will cache any asset it loads and wrap the asset in an [asyncWritable](/docs/reference/core/utilities#asyncwritable) for convenience. Caching assets will reduce network requests, bandwidth, and memory usage, which will improve the performance of your application.
```

File: apps/docs/src/content/learn/basics/loading-assets.mdx [hunk 2]
```diff
@@ -4,126 +4,35 @@ title: Loading Assets
...
 
 <Tip type="info">
-  This section assumes you placed your assets in your public folder or in a place in your
-  application where you can import them easily.
+  It is assumed you placed your assets in your public folder or in a place in your application where
```
Comment (michealparks) [inline apps/docs/src/content/learn/basics/loading-assets.mdx:10]: I'd recommend keeping "This section assumes" or "These instructions assumes". "It is assumed" is more ambiguous and I have a harder time understanding what is meant.

File: apps/docs/src/content/reference/extras/gltf.mdx [hunk 3]
```diff
@@ -42,15 +42,12 @@ componentSignature:
...
-
-Use the hook [`useGltf`](/docs/reference/extras/use-gltf) to reuse a glTF model or to use parts of it.
+`<GLTF />` loads a single model from a url. The `url` property is reactive and will load new 3D content if changed. The new content will be swapped as soon as loading is finished.
 
 <Tip type="tip">
-  See the hook [`useGltfAnimations`](/docs/reference/extras/use-gltf-animations) on how to use
-  animations of glTF Files.
+  Loading assets can be challenging, from textures to animations and multiple copies. The details on
+  our [loading assets](/docs/learn/basics/loading-assets) page should point you in the right
+  direction if you're stuck.
```
Comment (michealparks) [inline apps/docs/src/content/reference/extras/gltf.mdx:50]:
```suggestion
  Asset loading can sometimes produce surprising results. We've detailed common issues you may run into on our [loading assets](/docs/learn/basics/loading-assets) page.
```

File: apps/docs/src/content/learn/basics/loading-assets.mdx (hunk 2 above)
Comment (DefinitelyMaybe) [reply apps/docs/src/content/learn/basics/loading-assets.mdx:10]:
```suggestion
  This section assumes you placed your assets in your public folder or in a place in your application where
```

**Pull request merged**